HW03 Brock.Rmd

```diff
@@ -0,0 +1,26 @@
+---
+title: "COVID Mapping"
+author: "Brock Adams"
+date: "7/20/2020"
+output: html_document
+---
+
+```{r setup, include=FALSE}
+knitr::opts_chunk$set(echo = TRUE)
+library(readr) #reading rectangular text data
+library(ggplot2) #plotting data
+library(magrittr) #piping
+```
+
+```{r COVID Data import}
+COVID_url = "https://raw.githubusercontent.com/nytimes/covid-19-data/master/us-states.csv"
+COVID_data <- read_csv(url(COVID_url))
+COVID_CA <- subset(COVID_data, state == "California")
+```
+
+```{r COVID Data Plotting}
+COVID_CA %>% 
+ggplot(aes(x = as.Date(date), y = deaths)) + geom_line() +
+  xlim(as.Date("2020-03-01"), as.Date("2020-07-20"))+ 
+  xlab("Date") + ylab("Deaths")+ ggtitle("COVID-19 deaths over time in California")
+```
```

Comment:
I like how straight-forward your code and graph are! If given more time, it would be cool to see some annotations on the graph, potentially detailing when the state went into quarantine and when those restrictions were lifted/reinforced again. This could be done with what Robert had us do in HW02 with the "diamonds" graph, in which we used annotate() with argument geom = "text".